### PR TITLE
Hotfix for plotting bug

### PR DESCRIPTION
**Pull request by christinahedges**
Plots currently don't work because there is a tiny error in the way we implemented it. Currently they plot nothing. I'm fixing it.

### CODE REVIEW

File: lightkurve/lightcurve.py [hunk 1]
```diff
@@ -626,7 +626,7 @@ def _create_plot(self, method='plot', ax=None, normalize=True,
             elif method == 'errorbar':
                 ax.errorbar(x=self.time, y=flux, yerr=flux_err, **kwargs)
             else:
-                ax.plot(x=self.time, y=flux, **kwargs)
```
Comment (barentsen) [inline lightkurve/lightcurve.py]: Mea culpa!

**Comment by barentsen** [comment]
Thanks @christinahedges !

File: lightkurve/lightcurve.py (hunk 1 above)
Comment (christinahedges) [reply lightkurve/lightcurve.py]: Actually......That was my fault........

Comment (barentsen) [reply lightkurve/lightcurve.py]: :open_mouth:

**Pull request merged**